bucket_4D/xlockmore/manifests/plist.single

```diff
@@ -0,0 +1,8 @@
+@mode(4111) bin/play.sh
+@mode(4111) bin/xlock
```

Comment:
Yeah, I was probably wrong here, @mode() is probably not legal.
@(,,4111) is what we really want.  Sorry, my fault.

Reply:
Ok, I've noticed error at check-plist stage :)